src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/AssessmentOrgsXlsxService.cs

```diff
@@ -38,9 +40,24 @@ public AssessmentOrganisationsDTO GetAssessmentOrganisationsData()
 
                 using (ExcelPackage package = new ExcelPackage(new FileInfo(filePath)))
                 {
-                    GetAssessmentOrganisations(package, assessmentOrgs);
+                    assessmentOrgs = GetAssessmentOrganisations(package).ToList();
```

Comment:
There would be multiple enumerations (ie: multiple queries to ES from lines 46 and 55)